src/plugin/planned-notification/Manager/PlannedNotificationManager.php

```diff
@@ -157,7 +165,15 @@ public function generateScheduledTasks(
     public function sendMessages(array $messages, array $users)
     {
         foreach ($messages as $message) {
-            $this->mailManager->send($message->getTitle(), $message->getContent(), $users);
+            $this->dispatcher->dispatch(
+                MessageEvents::MESSAGE_SENDING,
+                SendMessageEvent::class,
+                [
+                    $message->getContent(),
+                    $message->getTitle(),
+                    [$users],
```

Comment:
Array of array. IMO the wrapping array must be removed.

Reply:
Oops my bad, fixed. Thx :)